dll/cpl/inetcpl/lang/He.rc

```diff
@@ -71,8 +71,10 @@ FONT 8, "MS Shell Dlg"
 CAPTION "מחק היסטוריית גלישה"
 BEGIN
 
+    DEFPUSHBUTTON  "Cancel", IDCANCEL, 185, 230, 60, 15, WS_GROUP
+    PUSHBUTTON     "Delete", IDOK, 120, 230, 60, 15
```

Comment:
Is that reordering inevitable to fix the bug? Usually we do aim to have the controls that are on the bottom of the dialog (high y coordinates) after the ones at the top of the dlg (low y coordinates).

Reply:
The cancel button is DEFPUSHBUTTON so Enter will push it even if it does not have focus. Where does Windows put the focus?

Reply:
Windows puts the focus on the cancel button so that when pressing space it closes, I tested the behavior when moving the button group back down and while it is true that enter closes it space changed the state of the checkbox instead of closing the menu

Reply:
Windows (11) also treats each checkbox as it's own group so that arrow keys can't be used to change focus to a different checkbox, this to me seemed stupid so I didn't went that route

Reply:
Do I understand your comment correctly?
```
We need that reordering-change to make our INETCPL behave correctly on Win 11.
We wouldn't need that reordering-change to make our INETCPL behave correctly on 2k3/XP.
```
Or did I misunderstood you?

Reply:
Ah my bad, I was talking about how the windows's own delete browser history menu works

Reply:
If I open our unpatched inetcpl on XPSP3, then the cancel button **is** focused by default. So what is the purpose of that reordering then?

Reply:
That's because the unpatched version doesn't have a group for the checkboxes, with the WS_GROUP added to the IDC_DELETE_TEMP_FILES the checkbox group gets the focus instead of the cancel button.

Reply:
Can you come up with a solution which avoids the reordering?

Reply:
I'll see what I can do, for the record this is what I'm talking about, in both occasions I'm just pressing space


https://github.com/reactos/reactos/assets/36992813/696fe5f1-5009-4f65-ab1f-15e6c1dd5ff0

Reply:
> Can you come up with a solution which avoids the reordering?

No.

@maj113 you're fine to continue as you are just ignore him.

Reply:
I'm not in favour of moving these buttons to the top: this is unnatural and looks like a hack.
Either accept that one of the radiobuttons is selected when opening the dialog, or ensure in the code that the cancel button is selected, or have at least the cancel button be the defpushbutton (which it already is).
This is the standard behaviour that is present in all other dialogs on the system.

Reply:
If there’s a separate issue here, that’s fine then say that.

 But not matching a decompilation of windows resource files 1:1 isn’t a hack. Let’s not pretend it is.

Reply:
In any other dialog, having the bottom buttons at the end in the dialog resource don't cause problems.
As presented in this PR (and this list of comments), it seems to cause an "issue" and for that they are placed at the top: this is why I'm saying it's a hack.

Reply:
Sounds good 👍